src/dao/pages/DAOsListPage/DAOsListPage.tsx

```diff
@@ -28,22 +70,74 @@ export function DAOsListPage() {
         />
       }
       body={
-        <CardGridLayout<DAO[]>
-          columns={{ md: 2, lg: 3 }}
-          queryResult={daos}
-          message={"It looks like no DAOs have been created yet."}
-          preLinkText={"Click on this link to&nbsp;"}
-          linkText={"create the first DAO"}
-          onLinkClick={handleLinkClick}
-        >
-          {daos.data?.map((dao: DAO, index: number) => {
-            const { accountEVMAddress, name, type, isPrivate, logoUrl } = dao;
-            if (isPrivate) return null;
-            return (
-              <DAOCard key={index} accountEVMAddress={accountEVMAddress} name={name} type={type} logoUrl={logoUrl} />
-            );
-          })}
-        </CardGridLayout>
+        <Flex direction="column" gap={4}>
+          <Tabs
+            defaultIndex={isMyDAOsTabSelected ? 1 : 0}
+            onChange={handleTabChange}
+            bg={Color.White}
+            variant="dao-dashboard-tab"
+          >
+            <TabList borderBottom="0">
+              <Tab>
+                <Box>All</Box>
+              </Tab>
+              <Tab>
+                <Box>My DAOs</Box>
+              </Tab>
+            </TabList>
+          </Tabs>
+          <Flex justifyContent="flex-end" gap={4}>
+            <Input
+              variant="filter"
+              value={searchText}
+              onChange={(e: ChangeEvent<HTMLInputElement>) => handleSearchTextChange(e.target.value)}
+              placeholder="Search"
+              width="fit-content"
+              minWidth="unset"
+            />
+            <Select
+              variant="formTokenSelector"
+              width="7rem"
+              placeholder="All"
+              value={daoType}
+              onChange={(e: ChangeEvent<HTMLSelectElement>) => handleDAOTypeChange(e.target.value)}
+            >
+              {Object.values(DAOType).map((option) => {
+                return (
+                  <option key={option} value={option}>
+                    {option}
+                  </option>
+                );
+              })}
+            </Select>
+          </Flex>
+          {paginatedDAOs?.length === 0 && !daos.isLoading && (
+            <NotFound message="No DAOs have been found with the filters selected" />
+          )}
```

Comment:
we can also confirm the messages from Product team.

Reply:
Done.